defender/staged/downloaded/Forta_cToken.js

```diff
@@ -135,11 +135,18 @@ exports.handler = async function (autotaskEvent) {
     cTokenSymbol,
     eventName,
     usdValue,
+    protocolVersion,
   } = metadata;
   if (usdValue === undefined) {
     throw new Error('usdValue undefined, please use newer bot version');
   }
 
+  // Handle older alerts which don't specify the protocol version
+  let versionString = '';
+  if (protocolVersion !== undefined) {
+    versionString = ` (Compound v${protocolVersion})`;
+  }
+
```

Comment:
This can be hardcoded to (Compound v2). The cToken Bot only checks market activity on Compound V2.